R/query-webgeom.R

```diff
@@ -21,7 +21,7 @@ setMethod(f = "query",signature("webgeom",'character'),
               }
               url <- sprintf('%s?service=WFS&version=%s&request=GetFeature&typename=%s&propertyname=%s',
                              url(.Object), version(.Object), geom(.Object), .Object@attribute)
-              input_list[['key']] = geom(.Object)
+              input_list[['key']] <- sprintf("%s/*[local-name()='%s']", geom(.Object), .Object@attribute)
```

Comment:
Can't say I'm entirely sure what I'm looking at here... But if this is grabbing the desired attribute explicitly rather than assuming that only the requested attribute came back, then 👍